src/client/components/Forms/Signup.style.css

```diff
@@ -0,0 +1,123 @@
+.github-logo {
+  width: 30px;
+}
+.button-forms {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+.github-container {
+  padding: 0.3em;
+  background-color: #ebebeb;
+  border-radius: 5px;
+  margin: 5px;
+}
+.space {
+  margin-left: 80px;
+}
+.or {
+  color: #9a9a9a;
+}
+
+.input {
+  width: 100%;
+  padding: 10px 22px;
+  margin: 10px 0px;
+  box-sizing: border-box;
+  border: 2px solid #f8f0f4;
+  display: inline-block;
+  background-color: rgba(63, 58, 58, 0.075);
+}
+
+.form {
+  height: 600px;
+  width: 400px;
+  margin-right: auto;
+  margin-left: auto;
+  margin-top: 0px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  padding: 0px;
+}
+.label {
+  font-size: 14px;
+
+  margin-top: 10px;
+  margin-left: 5px;
+  text-align: right;
+  font-weight: bold;
+}
+
+.input-checkbox {
+  text-decoration: none;
+  font-size: 8px;
+}
+
+h3 {
+  border-left: 190px solid hsl(240, 15%, 92%);
+  border-right: 190px solid #e6e5ee;
+  line-height: 1px;
+  color: #a17272;
+  margin-top: 40px;
+}
+
+.label-checkbox {
+  display: flex;
+  font-size: 10px;
+  justify-content: left;
+  text-align: left;
+  font-weight: normal;
+}
+.validation {
+  width: 700px;
+  border: 0px solid rgb(250, 252, 250);
+  padding: 80px;
+  margin: 10px;
+  text-align: center;
+  color: red;
+  margin-top: -250px;
+}
+
+@media screen and (max-width: 750px) {
+  .form {
+    width: 100%;
+  }
+  .input {
+    width: 100%;
```

Comment:
For everything inside the media query, just double check which properties change according to the classes you have. If the properties do not change from what you specify outside of the media query, you don't need to specify it again inside the media query as it will be inherited. Just write in the media query those properties that actually change.
Fx in this case of the .input, you have the exact same properties with the same values inside and outside the media query, you probably do not need them in the media query.